src/testdrive/src/action/kafka/ingest.rs

```diff
@@ -174,45 +191,79 @@ impl Action for IngestAction {
 
         let futs = FuturesUnordered::new();
         for row in &self.rows {
-            let mut buf = Vec::new();
+            let mut valbuf = Vec::new();
+            let mut keybuf = Vec::new();
```

Comment:
For consistency with naming of other key- and value-related variables in this function let's call these `key_buf` and `value_buf`?